ppdet/modeling/mot/tracker/base_jde_tracker.py

```diff
@@ -18,7 +18,6 @@
 import numpy as np
 from collections import deque, OrderedDict
 from ..matching import jde_matching as matching
-from ppdet.core.workspace import register, serializable
```

Comment:
Must keep `register` here. You can test  whether the `Training/Eval/Infer` can run successfully.

Reply:
Those two have been kept since they are decorators.